src/Microsoft.Win32.Registry/pkg/Microsoft.Win32.Registry.pkgproj

```diff
@@ -4,7 +4,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ref\Microsoft.Win32.Registry.csproj">
-      <SupportedFramework>net46;netcoreapp1.0</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1</SupportedFramework>
```

Comment:
I had changed the assembly version in src\Microsoft.Win32.Registry\dir.props, see:
https://github.com/dotnet/corefx/pull/12392/files#diff-32a533d887f88629df818f0d01e1db86R2